### PR TITLE
Convert userguide to asciidoc: nativeBinaries - playPlugin

### PR DESCRIPTION
**Pull request by wolfs**
Another one for #2740

### CODE REVIEW

File: subprojects/docs/src/docs/userguide/nativeBinaries.adoc [hunk 1]
```diff
@@ -0,0 +1,943 @@
...
+
+
+[[native-binaries:tool-chain-support]]
+=== Tool chain support
+
+Gradle offers the ability to execute the same build using different tool chains. When you build a native binary, Gradle will attempt to locate a tool chain installed on your machine that can build the binary. You can fine tune exactly how this works, see <<native_binaries:tool_chain>> for details.
+
+The following tool chains are supported:
+
+[cols="a,a,a", options="header"]
```
Comment (eriwen) [inline subprojects/docs/src/docs/userguide/nativeBinaries.adoc]: ⭕️  Nitpick: this causes different table layout than what we had before. 

#### Old
screen shot 2017-09-05 at 9 52 46 am

#### New 
screen shot 2017-09-05 at 9 52 36 am

Note that in general, Asciidoc seems to generate `<p>` tags within every table cell, adding unnecessary padding to the bottom. This should not be addressed here, but wanted to mention.

Comment (wolfs) [reply subprojects/docs/src/docs/userguide/nativeBinaries.adoc]: Any ideas how to fix that?

**Comment by wolfs** [comment]
@erwin I removed the two unrelated commits, sorry for that.
Are we good to merge?

**Comment by eriwen** [comment]
Yes

**Pull request merged**